### PR TITLE
adding oc client into dev container

### PR DESCRIPTION
**Pull request by shiywang**

### CODE REVIEW

File: .devcontainer/dev.Dockerfile [hunk 1]
```diff
@@ -29,6 +29,10 @@ RUN wget -O /etc/yum.repos.d/rcm-tools-fedora.repo https://download.devel.redhat
   && dnf install -y rhpkg \
   && dnf clean all
 
+#include latest oc client
+RUN wget -O /tmp/openshift-client-linux-4.2.4.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.2.4.tar.gz \
```
Comment (vfreex) [inline .devcontainer/dev.Dockerfile]: 1. You need to delete the tarball from /tmp after installation.
2. Would be better to have the version number parameterized.

File: .devcontainer/dev.Dockerfile [hunk 2]
```diff
@@ -29,6 +29,10 @@ RUN wget -O /etc/yum.repos.d/rcm-tools-fedora.repo https://download.devel.redhat
...
+  && tar -C /usr/local/bin -xzf  /tmp/openshift-client-linux-4.2.4.tar.gz
```
Comment (vfreex) [inline .devcontainer/dev.Dockerfile]: Use `tar -C /path -xzf foo.tar.gz file1 file2` to filter the binaries out.

**Comment by shiywang** [comment]
@vfreex PTAL

**Pull request merged**